extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchBufferAggregator.java

```diff
@@ -113,4 +110,28 @@ public void close()
     unions.clear();
   }
 
+  @Override
+  public void relocate(int oldPosition, int newPosition, ByteBuffer newBuffer)
```

Comment:
Keeping the original interface `relocate(int oldPosition, int new Position, ByteBuffer newBuffer)` b/c of https://github.com/druid-io/druid/pull/4071#discussion_r106768296